sources/api/migration/migrations/v1.12.0/oci-defaults-setting-metadata/src/main.rs

```diff
@@ -0,0 +1,28 @@
+#![deny(rust_2018_idioms)]
+use migration_helpers::common_migrations::{AddMetadataMigration, SettingMetadata};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We updated the 'affected-services' list metadata for 'settings.oci-defaults'
+/// to include itself and containerd on upgrade, and to remove those values on
+/// downgrade, depending on the running variant.
+fn run() -> Result<()> {
+    if cfg!(variant_runtime = "k8s") {
+        migrate(AddMetadataMigration(&[SettingMetadata {
+            metadata: &["affected-services"],
+            setting: "settings.oci-defaults",
+        }]))?;
+    };
```

Comment:
nit: I don't think that last semicolon is needed:

```suggestion
    if cfg!(variant_runtime = "k8s") {
        migrate(AddMetadataMigration(&[SettingMetadata {
            metadata: &["affected-services"],
            setting: "settings.oci-defaults",
        }]))?;
    }
```